packages/styles/scss/components/masthead/_masthead-megamenu.scss

```diff
@@ -203,42 +203,55 @@
     p,
     a,
     .#{$prefix}--link-with-icon {
-      padding: 6px $spacing-05;
-      text-decoration: none;
       width: 100%;
-
-      @include type-style('expressive-heading-01');
     }
 
+    p,
+    a,
+    .#{$prefix}--link-with-icon,
     ::slotted(span),
     span {
-      color: $text-01;
+      @include carbon--type-style(heading-03, true);
 
+      color: $text-02;
+      padding: 6px $spacing-05;
```

Comment:
Jinx! I left a similar review (https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/9267#discussion_r954263688). This "magic" number might be related to old specs of when the Global Masthead megamenu was developed a year ago. I would just leave it as is because `6` is not part of the available Carbon spacing token and work on converting the `px` value to `rem`.

Reply:
Bingo. I'll replace these magic numbers with `carbon--rem()` as recommended in the other review comment.